src/utils/document.js

```diff
@@ -339,3 +339,122 @@ export const getFeedbackCategoryLabel = (category) => {
   };
   return categories[category];
 };
+
+export const processedTableDocumentFields = (documents, sortKey, sortDirection) => {
```

Comment:
Generic naming ... Don't need to know this is being used on a table. If it's a sort function, name it so. Current naming tells me that this function is returning processed table document fields.